modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -877,8 +877,7 @@ class CV_EXPORTS_W VideoWriter
     @param filename Name of the output video file.
     @param fourcc 4-character code of codec used to compress the frames. For example,
     VideoWriter::fourcc('P','I','M','1') is a MPEG-1 codec, VideoWriter::fourcc('M','J','P','G') is a
-    motion-jpeg codec etc. List of codes can be obtained at [Video Codecs by
-    FOURCC](http://www.fourcc.org/codecs.php) page. FFMPEG backend with MP4 container natively uses
+    motion-jpeg codec etc. List of codes can be obtained at [MSDN](https://docs.microsoft.com/en-us/windows/win32/medfound/video-fourccs) page. FFMPEG backend with MP4 container natively uses
```

Comment:
What about adding both sources of info?

Something like:

```suggestion
    motion-jpeg codec etc. List of codes can be obtained at [MSDN](https://docs.microsoft.com/en-us/windows/win32/medfound/video-fourccs) page (see also this [archived page](https://web.archive.org/web/20220316062600/http://www.fourcc.org/codecs.php) of the fourcc site for a more complete list). FFMPEG backend with MP4 container natively uses
```

---

For instance, for some reason the Microsoft site does not mention the `MJPG` codec while it should be a quite common codec I think?
It is BTW mentioned in the `VideoWriter` doc:

>  VideoWriter::fourcc('M','J','P','G') is a motion-jpeg codec etc.

Reply:
I think I will let you guys decide how you want your docs, but if I may comment on this.

Since MSDN contains a subset of the other one it does not add much help to those who try to find their fourcc. Having two links means two links that might be changed or outdated. As far as I know does not microsoft guarantee that they will keep the same URLs. 

There are also several other sites that may be less reputable, but have a larger list of fourccs. E.g. [codecs.com](https://www.free-codecs.com/guides/fourcc.htm). I don't know who is behind that domain and I guess you prefer a well known source?